Update RecorderX.py

ProtectorGamer opened the pull request:
Updated some basic code language and made it in simplified. 

Recorder Thread Updated, Updated error-handling & Added docstrings

ProtectorGamer committed:
Update RecorderX.py

Updated some basic code language and made it in simplified. 

Recorder Thread Updated, Updated error-handling & Added docstrings

jindalpriyanshu101 commented:
Gone through your pull request, it looks good but require a few changes.

- Keep default `GUI` initializations, without that GUI will adapt system theme and cause an issue for white themed users:
```
ctk.set_appearance_mode("dark")
ctk.set_default_color_theme("blue")
```

- Revert the change of convention `RecorderX` to `recorder_thread`, we're not meant to change the main function name for the time being :)

jindalpriyanshu101 reviewed (changes requested):
Gone through your pull request, it looks good but require a few changes.

- Keep default `GUI` initializations, without that GUI will adapt system theme and cause an issue for white themed users:
```
ctk.set_appearance_mode("dark")
ctk.set_default_color_theme("blue")
```

- Revert the change of convention `RecorderX` to `recorder_thread`, we're not meant to change the main function name for the time being :)

ProtectorGamer committed:
Update RecorderX.py

Required lines has been added for GUI Configuration as per requirement by @jindalpriyanshu101

ProtectorGamer committed:
Update RecorderX.py

Updated Recorder_thread to RecorderX

jindalpriyanshu101 reviewed (approved):
Looks Good To Me! Approved.
